depends/packages/qt.mk

```diff
@@ -1,21 +1,21 @@
 PACKAGE=qt
-$(package)_version=5.7.1
-$(package)_download_path=http://download.qt.io/new_archive/qt/5.7/$($(package)_version)/submodules
```

Comment:
Will this fix CI?
`$(package)_download_path=https://download.qt.io/official_releases/qt/5.15/$($(package)_version)/submodules`

Reply:
I was *certain* I changed that. Good catch. Let's try it.

Reply:
Ah, I didn't run `make -j9 -C depends` locally, which will give me more information. I'll get back to this in a day or two.

Reply:
I think, in addition to the download paths changing, the format of the downloadable SHA-256 checksums has changed too:

https://download.qt.io/official_releases/qt/5.15/5.15.7/submodules/qtcharts-everywhere-opensource-src-5.15.7.tar.xz.sha256

I'm lost in a twisty maze of autoconf definitions right now though.